commands/info.md

```diff
@@ -233,6 +233,10 @@ Here is the meaning of all fields in the **stats** section:
 *   `expired_time_cap_reached_count`: The count of times that active expiry cycles have stopped early
 *   `expire_cycle_cpu_milliseconds`: The cumulative amount of time spend on active expiry cycles
 *   `evicted_keys`: Number of evicted keys due to `maxmemory` limit
+*   `total_eviction_exceeded_time`:  Total time used memory is greater than `maxmemory`
+     since server startup, in milliseconds
+*   `current_eviction_exceeded_time`: Duration currently used memory is greater than `maxmemory`,
+     if used memory is below `maxmemory` this metric is reset to 0, in milliseconds
```

Comment:
Maybe a slight rephrasing would be clearer and more concise?

```suggestion
*   `current_eviction_exceeded_time`: The number of milliseconds since used_memory was below maxmemory. 
```

Reply:
Should below -> above ?
You are native English speaker. Your description is better. My English is unskilled.

Reply:
"since used_memory was below" == "since used_memory is above", so the @madolson's phrasing is good.

Reply:
I think "since used_memory is above" is more clear.

Reply:
how about this then?
```suggestion
* `current_eviction_exceeded_time`: The number of milliseconds since `used_memory` grew above `maxmemory`.
```

Reply:
Good.